libevm/EVMC.cpp

```diff
@@ -15,10 +15,13 @@ EVM::EVM(evmc_instance* _instance) noexcept : m_instance(_instance)
     assert(m_instance != nullptr);
     assert(m_instance->abi_version == EVMC_ABI_VERSION);
 
+    cnote << "EVMC name=" << m_instance->name << " version=" << m_instance->version;
```

Comment:
i've found an issue. The log cannot be here, because it will be shown for every transaction. You can change it that EVM wrapper is created only once.

Reply:
That might be too big of a work for me in aleth, will just remove the message. Unless you want to push a fix yourself?

Reply:
Created an issue to track this https://github.com/ethereum/aleth/issues/5223.